src/components/common/GitInfoMaterial.tsx

```diff
@@ -273,7 +287,7 @@ export default function GitInfoMaterial({
                 ) : (
                     <>
                         {renderMaterialSource(context)}
-                        {renderMaterialHistory(context, selectedMaterial)}
+                        {renderMaterialHistory(context, selectedMaterial ?? material)}
```

Comment:
Is this intentional? Do we need to assign some default if selectedMaterial is not present?

Reply:
Confirm if we have any state when selectedMaterial is not present

Reply:
we don't have any state when a selectedMaterial is not present, hence this intentional and required